crates/datastore/src/redis/redis_cache.rs

```diff
@@ -102,7 +125,31 @@ impl RedisCache {
         tokio::spawn(async move { while let Ok(_message) = pa_recv.recv().await {} });
         tokio::spawn(async move { while let Ok(_message) = he_recv.recv().await {} });
 
-        let cache = Self { pool, tx, inclusion_list, payload_attributes, head_event };
+        let top_bid_value_cache =
+            Cache::builder().time_to_idle(Duration::from_secs(45)).max_capacity(10_000).build();
+        let floor_bid_value_cache =
+            Cache::builder().time_to_idle(Duration::from_secs(45)).max_capacity(10_000).build();
+        let seen_block_hashes =
+            Cache::builder().time_to_idle(Duration::from_secs(45)).max_capacity(10_000).build();
+        let builder_latest_payload_received_at =
+            Cache::builder().time_to_idle(Duration::from_secs(45)).max_capacity(10_000).build();
+        let builder_info_cache =
+            Cache::builder().time_to_idle(Duration::from_secs(300)).max_capacity(10_000).build();
```

Comment:
do we expect builder info to always be fetched or updated every 5 minutes? is there a danger that an entry will get expired as idle incorrectly?